Update to OpenShift FAQs for G11 (#613)
UKCloud/documentation: articles/openshift/oshift-faq.md

### What is UKCloud for OpenShift?

> UKCloud for OpenShift is a Kubernetes based Platform as a Service (PaaS) solution providing container management and orchestration using either Red Hat OpenShift to deliver a flexible, scalable cloud application platform. Unlike traditional managed PaaS offerings, it provides a modern application platform that accelerates end-to-end development, deployment and operation of digital applications, while raising overall application reliability and availability.

Reviewer: It says "using either Red Hat OpenShift" but there's no "or"...

Reviewer 2: ```suggestion
UKCloud for OpenShift is a Kubernetes based Platform as a Service (PaaS) solution providing container management and orchestration using Red Hat OpenShift to deliver a flexible, scalable cloud application platform. Unlike traditional managed PaaS offerings, it provides a modern application platform that accelerates end-to-end development, deployment and operation of digital applications, while raising overall application reliability and availability.
```

### Is UKCloud for OpenShift a single-tenant or multi-tenant solution?

UKCloud for OpenShift is built as an isolated single-tenant platform on-top of UKCloud's secure, assured multi-tenant UKCloud for OpenStack IaaS platform, helping to deliver the benefits of single tenant isolation with the economics and flexibility of multi-tenant infrastructure.

### How is UKCloud for OpenShift billed?

This service comprises of two main chargeable elements:

  - **Foundation Pack -** providing an initial footprint of 32GiB of RAM, billed by the month with a one-month minimum commitment

>   - **Runtime Pack -** billed by the month with a one-month minimum commitment based upon the amour of additional RAM consumed

Reviewer: Add an empty line between the two bullet points so that they render correctly in the generated HTML

Reviewer 2: Also this isn't correct - the foundation pack provides the control plane and infrastructure nodes - 32GiB of RAM suggests customers can run their applications on that 32GiB which they can't.

Reviewer 2: ```suggestion

  - **Runtime Pack -** billed by the month with a one-month minimum commitment based upon the amount of additional RAM allocated
```

### Is UKCloud for OpenShift a single-tenant or multi-tenant solution?

UKCloud for OpenShift is built as an isolated single-tenant platform on top of UKCloud's secure, assured multi-tenant UKCloud for OpenStack IaaS platform, helping to deliver the benefits of single-tenant isolation with the economics and flexibility of multi-tenant infrastructure.

### How is UKCloud for OpenShift billed?

This service comprises of two main chargeable elements:

>   - **Foundation Pack -** providing an initial footprint of 32GiB of RAM, billed by the month with a one-month minimum commitment

Reviewer 2: I'm still not happy with this bit. I've made a suggestion below though - but it needs to clarify that the foundation pack doesn't provide any resource to run applications on. Only the control plane and infrastructure services.

```suggestion
  - **Foundation Pack -** providing the cluster control plane and an initial footprint of 32GiB of RAM to provide the infrastructure services, billed by the month with a one-month minimum commitment. The addition of Runtime packs is required to provide the resource to run customer applications.
```

### Can I use UKCloud for OpenShift in the UKCloud Elevated (previously IL3) domain?

> UKCloud for OpenShift services are available in both our Assured (previously IL2) and upon request in our Elevated (previously IL3) security domains.

Reviewer: I think we can get rid of the previously IL2/IL3 bit now

Author: Updated accordingly

### Can I use UKCloud for OpenShift over closed networks such as PSN and HSCN?

> UKCloud for OpenShift has been certified for use over both the PSN Assured and Protected networks.

Reviewer: I don't think we should be referring to PSN Assured and PSN Protected anymore now that the network has merged it's all just PSN. So may "certified for use over the PSN network in both our Assured and Elevated security domains."

Author: Updated accordingly

> Connectivity to the N3 network is available for customers and partners serving the healthcare community.

Reviewer: Change N3 to HSCN

Author: Updated accordingly
